code/game/mecha/mecha.dm

```diff
@@ -2084,6 +2088,38 @@
 	return "huddiagmax"
 
 
+/obj/item/device/mech_painter
+	name = "Mecha Painter"
```

Comment:
lowercase, also don't forget to add an R&D design or however this is gonna be made